test/acr_cloud_response_test.dart

```diff
@@ -0,0 +1,216 @@
+import 'package:flutter_acrcloud/src/acrcloud_response.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  test(
+    'Should parse music JSON metadata',
+    () {
+      final response = ACRCloudResponse.fromJson(mockMusicFilesData);
+
+      final data = response.metadata?.music?.first;
```

Comment:
For all of these tests, you expect the data to be present. So you can just use `!` instead of `?`, and if it were to fail, then the test would fail as you'd want it to.